airbyte-server/readme.md

```diff
@@ -0,0 +1,3 @@
+# airbyte-server
+
+This module contains the actual app that runs the Airbyte Configuration API. The main method can be found in `ServerApp.java`. The external API interface that it implements is declared in `airbyte-api`.
```

Comment:
I think it worth mentioning that many handlers that interact with the persistence layer are present in this module.

Reply:
good point. added it.